app/src/main/java/org/p2p/wallet/home/ui/main/bottomsheet/ActionsBottomSheet.kt

```diff
@@ -0,0 +1,84 @@
+package org.p2p.wallet.home.ui.main.bottomsheet
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.os.bundleOf
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.setFragmentResult
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import org.p2p.wallet.R
+import org.p2p.wallet.databinding.DialogMainActionsBinding
+import org.p2p.wallet.databinding.ViewActionItemBinding
+import org.p2p.wallet.utils.args
+import org.p2p.wallet.utils.viewbinding.viewBinding
+import org.p2p.wallet.utils.withArgs
+
+private const val EXTRA_REQUEST_KEY = "EXTRA_REQUEST_KEY"
+private const val EXTRA_RESULT_KEY = "EXTRA_RESULT_KEY"
+
+class MainActionsBottomSheet : BottomSheetDialogFragment() {
+
+    companion object {
+        fun show(
+            fm: FragmentManager,
+            requestKey: String,
+            resultKey: String
+        ) = MainActionsBottomSheet().withArgs(
+            EXTRA_REQUEST_KEY to requestKey,
+            EXTRA_RESULT_KEY to resultKey
+        ).show(fm, MainActionsBottomSheet::javaClass.name)
+    }
+
+    private val resultKey: String by args(EXTRA_RESULT_KEY)
+    private val requestKey: String by args(EXTRA_REQUEST_KEY)
+
+    private val binding: DialogMainActionsBinding by viewBinding()
```

Comment:
layout inflates two times here I think
in by viewBinding()
and in inflater.inflate

Reply:
yes but do you know what we need to do here?

Reply:
override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View = binding.root ?

Reply:
binding.root  - crash =)